AGPs/AGP-Aracon2020.md

```diff
@@ -0,0 +1,80 @@
+---
+AGP: N/A
+Title: Aracon 2020
+Author: @stefanobernardi
+Status: Stage III
+Track: Finance
+Created: 2019-10-01
+---
+
+# AGP-AC: AraCon 2020
+
+## Address of the transfer recipient
+
+- An ad-hoc AraCon DAO, or the Events DAO. The Aragon Association will provide an address.
```

Comment:
When will the address be provided?

(It's a bit of a contrived question because this is the AA giving itself money to do this, but just wondering if you are planning to add the address to the proposal)

Reply:
We don't have one as of now, as the members would still be in flux. As you state, given that it's the AA shifting its own resources, it makes sense to wait and make sure that the DAO that gets deployed is perfect - so if possible, we'd provide an address in the coming weeks.